### PR TITLE
Added compile hook to write SHA to disk

### LINKED ISSUE #81: Version listed on deployments
It would be nice if we could hit an endpoint (or it was written into the base template or admin or something) to get the current version deployed. This would help with validating that a particular change was on ccxcon-ci, for instance.

### PR DESCRIPTION
**Pull request by bdero**
Closes #81 !

### CODE REVIEW

**Comment by bdero** [comment]
Going to quickly deploy this as a test.

**Comment by bdero** [comment]
Also a good chance to test automated "review" environments.

File: bin/pre_compile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Syntax bin/compile <build-dir> <cache-dir>
+
+echo "-----> Writing out SOURCE_VERSION to static/hash ($SOURCE_VERSION)"
+echo $SOURCE_VERSION > $BUILD_DIR/staticfiles/hash
```
Comment (justinabrahms) [inline bin/pre_compile]: Can you change this to something like `hash.html`? Right now it just downloads the thing which is less than ideal.

**Comment by justinabrahms** [comment]
Not sure what you changed, but it worked before. Just needed the .html file extension.

**Comment by bdero** [comment]
@justinabrahms I just fixed this up, sorry for the delay - does it display in your browser?: https://ccxcon-ci-pr-84.herokuapp.com/static/hash.txt

**Comment by bdero** [comment]
Also, to answer your question earlier (since I don't think I really answered it). To invoke a build on any heroku environments launched for PRs, you can just push to the branch of the PR and it'll happen.

**Comment by justinabrahms** [comment]
:+1:

**Pull request merged**